hummingbot/connector/derivative/perpetual_finance/perpetual_finance_derivative.py

```diff
@@ -45,7 +47,7 @@
 logging.basicConfig(level=METRICS_LOG_LEVEL)
 
 
-class PerpetualFinanceDerivative(DerivativeBase):
+class PerpetualFinanceDerivative(ConnectorBase, PerpetualTrading):
```

Comment:
Are we going to start using multiple inheritance? It is something complex to deal with and could make things difficult to understand in the future.